File: babies_mod/UseCard.lua
local UseCard = {}

-- Includes
local PostRender = require("babies_mod/postrender")

-- Card.CARD_EMPRESS (4)
function UseCard:Card4(card)
  -- Using an Empress card will give us the Whore of Babylon costume,
  -- so we have to update the sprite
  PostRender:SetPlayerSprite()
end

-- Card.CARD_HANGED_MAN (13)
function UseCard:Card13(card)
  -- Using a Hanged Man card gives us flight, so we have to update the sprite
  PostRender:SetPlayerSprite()
end

return UseCard

File: src/callbacks/useCard.ts
import * as postRender from "./postRender";

export function init(mod: Mod): void {
  mod.AddCallback(ModCallbacks.MC_USE_CARD, empress, Card.CARD_EMPRESS); // 4
  mod.AddCallback(ModCallbacks.MC_USE_CARD, hangedMan, Card.CARD_HANGED_MAN); // 5
}

// Card.CARD_EMPRESS (4)
function empress() {
  // Using an Empress card will give us the Whore of Babylon costume,
  // so we have to update the sprite
  postRender.setPlayerSprite();
}

// Card.CARD_HANGED_MAN (5)
function hangedMan() {
  // Using a Hanged Man card gives us flight, so we have to update the sprite
  postRender.setPlayerSprite();
}
